pages/index.js

```diff
@@ -1,11 +1,28 @@
 import dynamic from 'next/dynamic';
+
 //import CardGrid from '../src/components/CardGrid/CardGrid';
```

Comment:
Please remove the comments you don't need.

Reply:
done